index.html

```diff
@@ -0,0 +1,153 @@
+          </div>
+        </nav>
+      </div>
+      <div class="row">
+        <nav class="navbar" id="navbar">
+          <div class="navbar-header">
+            <a class="navbar-brand" href="#">
+              <img src="assets/images/img4.png" alt="navbar logo imag" />
+            </a>
+          </div>
+          <ul class="nav navbar-nav navItem text-secondary" id="navMenu">
+            <li class="active"><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#program">Program</a></li>
+            <li><a href="#sports">Sports</a></li>
+            <li><a href="#">Sponser</a></li>
+            <li><a href="#">Nutrition</a></li>
+          </ul>
+
+          <div class="humgurger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        </nav>
+      </div>
+      <main class="main1">
+      <div class="row introduction">
+        <p>"Welcome to Healthline Nutrition!"</p>
+        <h1 class="secondHeader">
+          The Definitive Guide<br />
+          to Healthy Eating in Real Life
+        </h1>
+
+        <p id="second-p">
+          Healthy eating doesn’t have to be complicated. It’s entirely possible
+          to nourish your body while enjoying the foods you love.
+        </p>
+        <h3>2015.10.15(THU) ~ 16(FRI)</h3>
+        <h5>@ Do not focus on how much you eat ,Focus on what you eat .</h5>
+      </div>
+      <div class="row mainProgram" id="program">
+        <h3>Main Program</h3>
+        <hr />
+        <div class="col-sm card">
+          <img src="assets/images/vegtables.jpg" alt="Vegtables" />
+          <h4 class="card-title">Vegetables</h4>
+          <p class="card-text">
+            Add variety to grilled or steamed vegetables with an herb such as
+            rosemary..
+          </p>
+        </div>
+        <div class="col-sm card">
+          <img src="assets/images/fruits.jpg" alt=" fruits" />
+          <h4 class="card-title">Fruits</h4>
+          <p class="card-text">
+            Fresh, frozen, or canned fruits are great choices. Try fruits beyond
+            apples and bananas such as mango, pineapple or kiwi fruit
+          </p>
+        </div>
+        <div class="col-sm card">
+          <img src="assets/images/calcium.jpg" alt=" calcuim-rich food image" />
+          <h4 class="card-title">calcium-Rich food</h4>
+          <p class="card-text">
+            In addition to fat-free and low-fat milk, consider low-fat and
+            fat-free yogurts without added sugars.
+          </p>
+        </div>
+        <div class="col-sm card">
+          <img src="assets/images/meat.jpg" alt=" meat picture" />
+          <h4 class="card-title">Meat</h4>
+          <p class="card-text">
+            If your favorite recipe calls for frying fish or breaded chicken,
+            try healthier variations by baking or grilling.
+          </p>
+        </div>
```

Comment:
As mentioned by the previous reviewer, kindly add a fifth element to the list. Also kindly adjust the design so that all the elements are in a straight horizontal line 🤗
  - _The first element is out of order in your current design as shown below._

Your design:

![image](https://user-images.githubusercontent.com/73714615/190339787-3a5b399b-2be2-441d-80e2-af371d0b3e1f.png)

Template design:

![image](https://user-images.githubusercontent.com/73714615/190339836-f6004547-417f-4986-9e56-8bc0ebbb9699.png)

Reply:
> Add new card to be 5 cards.

> Fix the  size of first card

>